### PR TITLE
Newjersey eb

### PR DESCRIPTION
**Pull request by boppana**
This pull request aaddresses issue #new issue

### CODE REVIEW

**Comment by jdavis1998** [comment]
What is the best place to visit in Jersey that all the locals love  :heart: to go to?

**Comment by boppana** [comment]
In summer:
 beaches, boardwalk

**Comment by Govin-lala** [comment]
Very nice

File: githubteachergit-newjersey.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+checklist
+
+[ ]  one 
+
+###Best places :
+
+boardwalk
+delaware tubing
+
```
Comment (boppana) [inline githubteachergit-newjersey.md:18]: @jdavis1998

**Comment by boppana** [comment]
close issue #179

**Pull request merged**